native/src/components/TtsContainer.tsx

```diff
@@ -138,9 +138,11 @@ const TtsContainer = ({ children }: TtsContainerProps): ReactElement => {
   const updateSentences = useCallback(
     (newSentences: string[]) => {
       setSentences(newSentences)
-      stop()
+      if (isPlaying) {
+        stop()
+      }
     },
-    [stop],
+    [stop, isPlaying],
```

Comment:
```suggestion
    // eslint-disable-next-line react-hooks/exhaustive-deps
    [stop],
```

Reply:
:-1: Then this will always call stop or never call stop depending whether `isPlaying` is true or false the first time this is run.